tower_cli/resources/job_template.py

```diff
@@ -99,17 +99,12 @@ class Resource(models.SurveyResource):
         help_text='On write commands, perform extra POST to the '
                   'survey_spec endpoint.')
 
-    @resources.command
-    def create(self, fail_on_found=False, force_on_exists=False,
-               extra_vars=None, **kwargs):
-        """Create a job template."""
+    def write(self, *args, **kwargs):
         # Provide a default value for job_type, but only in creation of JT
-        if not kwargs.get('job_type', False):
+        if (kwargs.get('create_on_missing', False) and
```

Comment:
We force `job_type` to be "run", because tower-cli actually doesn't have a good system of defaults for fields. If we just set `default="run"` for the job_type field, then on any modify command it will _change_ the job_type to run, which is extremely undesirable. We want to interpret this kind of default as only applying when creating a new resource. This can be done with by the create command, or by the write command with --create-on-missing set.